README.md

```diff
@@ -91,6 +92,10 @@ You can pass the following options to generate a PDF:
     logo: "https://www.ruby-lang.org/images/header-ruby-logo@2x.png" # Downloaded with OpenURI
     ```
 
+  * `logo_fallback` - String to be displayed in place of a logo if no logo was specified. - _Optional, defaults to company name_
```

Comment:
What if we allow the `logo` option to just be a string? 

We can check if the `logo` includes `://` which would hint that it's a URL of some sort. If it doesn't include that, we can just render out the string. That way we don't have to introduce a new option?